listenbrainz/spark/spark_dataset.py

```diff
@@ -102,6 +111,85 @@ def handle_end(self, message):
 CouchDbDataset = _CouchDbDataset()
 
 
+class _StatsDataset(SparkDataset):
+
+    def __init__(self, stats_type):
+        super().__init__(stats_type)
+        self.workers = 2
```

Comment:
Should this be a const with comments as to how/when to change this for tuning purposes?

Reply:
Sure, I'll add a comment. FWIW, 2 threads is the best at the moment. The bottleneck shifts to spark at that point.